docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc

```diff
@@ -53,27 +53,50 @@ include::example$generated-documentation/apoc.create.virtualPath.adoc[]
 
 .Virtual node and virtual relationship `vNode`, `vRelationship`
 
+From this simple dataset:
+
+[source,cypher]
+----
+CREATE(a:Person)-[r:ACTED_IN]->(b:Movie)
+----
+
+we can do:
```

Comment:
How does the above example code tie to the below example code? Is there something I'm missing?

Reply:
I'm sorry, very bad copy paste mistake, I forget to change it :(